skins/10h16/templates/Donation_Form_Personal_Data.html.twig

```diff
@@ -365,6 +365,7 @@
                             <label for="street">Stra&szlig;e</label>
                             <input id="street" name="street" type="text" value="{$ street $}" data-pattern="^((?![M|m]uster).)+$" placeholder="z.B. Musterstra&szlig;e 123" class="required">
                             <span class="validation icon-placeholder"></span>
+                            <p id="street-warning-box" style="margin-left: 180px; margin-top: 10px; color: darkred; display: none">Ist die Adresse korrekt oder fehlt noch Ihre Hausnummer?</p>
```

Comment:
We try to avoid inline styles, as they are all possible blockers for introducing [Content Security Policy](https://en.wikipedia.org/wiki/Content_Security_Policy) in the future. Here, the styles can be moved to a class and a rule in the stylesheet.